common/cluster/metadata.go

```diff
@@ -227,11 +227,12 @@ func (m *metadataImpl) Start() {
 		return
 	}
 
-	err := m.refreshClusterMetadata(context.Background())
+	ctx := headers.SetCallerInfo(context.TODO(), headers.NewCallerInfo(headers.CallerTypeBackground))
```

Comment:
use context.Background() instead of TODO() unless you want to change it later to something else.

Reply:
Yeah, I think we might want to specify some timeout for the context so used TODO() here. let me add a comment.